src/frontend/components/SøknadsSteg/OmDeg/OmDeg.tsx

```diff
@@ -124,11 +124,13 @@ const OmDeg: React.FC = () => {
                     </KomponentGruppe>
                 )}
             </KomponentGruppe>
-            <UtenlandsoppholdModal
-                erÅpen={erÅpen}
-                toggleModal={toggleModal}
-                onLeggTilUtenlandsperiode={leggTilUtenlandsperiode}
-            />
+            {erÅpen && (
```

Comment:
Kanskje vi kunne gått for en konsekvent ordlyd på disse? enten erÅpen eller erUtenlandsoppholdmodalÅpen slik det var på noen av de andre? 😊

Reply:
Helt enig! Fikser